### PR TITLE
Hl student UI update

### PR DESCRIPTION
**Pull request by JayBae9903**
Here's what I did:

1. Set the instructions box to open by default
2. Disable input for the first applicant field.
3. Add validation for the preferred hall, and preferences (I tried to add the validation function for email, but I think it's more like a backend function, so I couldn't complete it. So, right now, the validation only checks if the email ends with @gordon.edu.)
4. Add submit completion snack bar
5. Minor changes

### CODE REVIEW

File: src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx:1]: The first and last name for applicant are no longer be used.

**Review by andrew-wzj — COMMENTED** [review]
Remove the code for first and last name

File: src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx
Comment (JayBae9903) [reply src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx:1]: I think Eric did that part. We can check when we combine branches.

**Comment by JayBae9903** [comment]
I added an email auto-complete function too. So, this branch has some minor improvements in the student UI.

**Comment by ahwnsals1234** [comment]
As for the 3rd thing you did, that's exactly what I did couple months ago and we already have it implemented. if you weren't able to do full validation of the email, can you just remove the fixes you made for that function?

**Comment by JayBae9903** [comment]
> As for the 3rd thing you did, that's exactly what I did couple months ago and we already have it implemented. if you weren't able to do full validation of the email, can you just remove the fixes you made for that function?

The problem I found was when I tried to submit the application with invalid emails, I was able to submit it. So I think the email validation function is not working now.

File: src/views/HousingLottery/adminView/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/adminView/index.jsx:1]: align the code format, approved

File: src/views/HousingLottery/studentView/Instructions/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/Instructions/index.jsx:1]: I already made a few changes on the instruction page by removing some unnecessary text

File: src/views/HousingLottery/studentView/Instructions/index.jsx [hunk 1]
```diff
@@ -1,121 +1,173 @@
...
-            <strong>4. How will I learn my room assignment?</strong><br/>
-            You will receive your room assignment via email no later than Friday, Apr.28.
+            <strong>
+              3. Complete questionnaire by providing name(s), email(s), and housing preferences by
+              Friday, Apr. 21 at noon.
+            </strong>
           </Typography>
+          <br />
           <Typography variant="body1" paragraph>
-            <strong>5. Do I need to be registered for classes to complete the questionnaire?</strong><br/>
```
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/Instructions/index.jsx:59]: Necessary text change starting from here

File: src/views/HousingLottery/studentView/Instructions/index.jsx [hunk 2]
```diff
@@ -1,121 +1,173 @@
...
+            email Housing on or after July 1 to request that.
+          </Typography>
+          <Typography variant="body1" paragraph>
+            <strong>
+              17. How can I receive confirmation that my questionnaire has been received?
+            </strong>
+            <br />
+            If you would like confirmation that your questionnaire has been received, you will need
+            to select the “Send me an email receipt of my responses” box at the bottom of the
+            questionnaire. No other confirmation will be provided.
```
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/Instructions/index.jsx:170]: Keep a note here to reserve the text change until here

File: src/views/HousingLottery/studentView/PreferenceBox/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/PreferenceBox/index.jsx:1]: why you think here should to have a new variable to store the preference option as newloudorQuiet? I don't think it's neceesary

File: src/views/HousingLottery/studentView/PreferenceBox/index.jsx [hunk 3]
```diff
@@ -73,6 +73,11 @@ const Preference = ({ onPreferenceChange }) => {
     }
   }, []);
 
+  useEffect(() => {
```
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/PreferenceBox/index.jsx:76]: Why add a new useEffect here to check valid for selected option?

File: src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx:1]: Approve the change here

File: src/views/HousingLottery/studentView/StudentApplicants/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/StudentApplicants/index.jsx:1]: approved

File: src/views/HousingLottery/studentView/PreferredHall/index.jsx
Comment (andrew-wzj) [inline src/views/HousingLottery/studentView/PreferredHall/index.jsx:1]: don't save the code change here

**Review by andrew-wzj — COMMENTED** [review]
Jay, please read my code review carefully. Since there are a few changed we already made in the senior project which causing your PR has conflicts with the current senior-project, I suggest you to create a new branch based on the senior-project and bring the code changes I marked as necessary into the new PR please. Thanks for your work

**Pull request closed without merging**